# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost-restored.html
# Commit: 8e0aea33f622
# Full Hash: 8e0aea33f622a76c5c8baa7dcf4485ffefd1662f
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2024-07-09 09:45:28
# Regressor Bug: 1892542
# File Overlap Count: 1
# Description:
#   Bug 1892542 - Webgl e.g. createBuffer() made infallible. r=gfx-reviewers,webidl,saschanaz,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D208050
# ==============================================================================

diff -r b953e0ccb68b -r 8e0aea33f622 dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost-restored.html
--- a/dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost-restored.html	Mon Jul 08 19:36:12 2024 +0300
+++ b/dom/canvas/test/webgl-conf/checkout/conformance/offscreencanvas/context-lost-restored.html	Mon Jul 08 17:24:12 2024 +0000
@@ -43,12 +43,12 @@
                 finishTest();
                 return;
             }
-            testLosingAndRestoringContext().then(function() {
+            testLosingAndRestoringContext().then(function(s) {
                 testPassed("Test passed");
                 finishTest();
                 return;
-            }, function() {
-                testFailed("Some test failed");
+            }, function(s) {
+                testFailed("Test failed: " + s);
                 finishTest();
                 return;
             });